### ISSUE TITLE
fsdp error 

### ISSUE THREAD

**Issue by charisliu123** [h2oai/h2ogpt#671]
i run the finetune.py without lora and use the fsdp ,but there is a error, fsdp/_init_utils.py,line889 inconsistent compute device and'device_id ' on rank 3: cuda:0 vs cuda:3; i run the programmer in Single machine multi card

**Comment by arnocandel** [comment]
Yes, WIP: https://github.com/h2oai/h2ogpt/pull/644/, known regression from the early days.
